src/zenml/integrations/huggingface/steps/accelerate_runner.py

```diff
@@ -116,11 +92,17 @@ def inner(*args: Any, **kwargs: Any) -> Any:
                             commands += [f"--{k}", f"{each}"]
                     else:
                         commands += [f"--{k}", f"{v}"]
-
                 logger.debug(commands)
 
                 parser = launch_command_parser()
                 args = parser.parse_args(commands)
+                for k, v in accelerate_kwargs.items():
+                    if k in args:
+                        setattr(args, k, v)
+                    else:
+                        logger.warning(
+                            f"You passed in Accelerate argument `{k}`, but it is not accepted by Accelerate."
```

Comment:
Its actually arguments to the accelerate launch command right? `accelerate launch...` .. not Accelerate generally?

Reply:
Also maybe link to https://huggingface.co/docs/accelerate/en/package_reference/cli#accelerate-launch to see accepted args

Reply:
Reworked